packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -978,6 +993,13 @@ abstract class SliderComponentShape {
   /// [labelPainter] already has the [textDirection] set.
   ///
   /// [value] is the current parametric value (from 0.0 to 1.0) of the slider.
+  ///
+  /// [textScaleFactor] can be used to determine whether the component should
+  /// paint larger or smaller, depending on whether [textScaleFactor] is greater
```

Comment:
I think it would be worth pointing out that textScaleFactor is typically the value of https://api.flutter.dev/flutter/widgets/MediaQueryData/textScaleFactor.html.  Note that this comment makes it sound like it could be less than or equal to zero. I don't think we want to support that; should assert that textScaleFactor is `>= 0`.

Here and elsewhere.

In the MediaQueryData docs we should probably explain that it's a positive value that's usually between 1 and 3 or 4, depending on the target platform. That can happen in a separate PR.